media/css/mozorg/contribute/taskview.less

```diff
@@ -435,3 +403,64 @@ a[role="button"] {
 .hidden {
     display: none;
 }
+
+/* newsletter module styles */
+.newsletter-module {
```

Comment:
Could include `contribute/newsletter.less` in the bundle instead of repeating code here?